packages/api/model/model.py

```diff
@@ -1,14 +1,15 @@
 import numpy as np
 import pandas as pd
 import copy
+import json
 from sklearn.preprocessing import MinMaxScaler
 from sklearn.decomposition import PCA
 from sklearn.ensemble import IsolationForest
 from sklearn.svm import OneClassSVM
 from sklearn.neighbors import LocalOutlierFactor
 from sklearn.utils import resample
 
-df = pd.read_csv("data/adani.csv")
+df = pd.read_csv("/home/josu/Desktop/Projects/StockGuardian/packages/api/model/data/adani.csv")
```

Comment:
Is it required to provide the full path? Relative path works if you run the model from the correct directory